### DISCUSSION TITLE
please help urgently

### DISCUSSION THREAD

**Discussion by cyberturkeyyy** [community/community#144355] [New to GitHub]
Hello,

The addresses below provide redirects to phishing websites. Can you help me to close the accounts? Redirection is provided only from Turkey, the sites are not accessible to all other countries.

https://github.com/ever2133-es/bets10-resmi
https://github.com/aarox4/bets10?tab=readme-ov-file

Thank you

**Comment by ghostinhershell** [comment]
Hi @cyberturkeyyy! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
